--- UaNet/src/train.py	2024-07-13 12:59:49.218792720 +0330
+++ UaNet/src/train.py	2024-07-14 10:34:41.505755975 +0330
@@ -223,6 +223,7 @@
 
         torch.cuda.empty_cache()
 
+    rpn_stats = torch.tensor(rpn_stats, dtype=torch.float32).cpu()
     rpn_stats = np.asarray(rpn_stats, np.float32)
     
     print('Train Epoch %d, iter %d, total time %f, loss %f' % (epoch, j, time.time() - s, np.average(total_loss)))
@@ -322,6 +323,7 @@
         rcnn_stats.append(rcnn_stat)
         mask_stats.append(mask_stat)
 
+    rpn_stats = torch.tensor(rpn_stats, dtype=torch.float32).cpu()
     rpn_stats = np.asarray(rpn_stats, np.float32)
     print('Val Epoch %d, iter %d, total time %f, loss %f' % (epoch, j, time.time()-s, np.average(total_loss)))
     print('rpn_cls %f, rpn_reg %f, rcnn_cls %f, rcnn_reg %f, mask_loss %f' % \
